source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -22,47 +22,16 @@ operators to modify an array embedded within a document:
 - :ref:`All Positional Operator <all-match-operator>`: ``$[]``
 - :ref:`Filtered Positional Operator <filtered-positional-operator>`: ``$[<identifier>]``
 
-See the MongoDB server guide on
-:manual:`Update Operators </reference/operator/update-array/#update-operators>`
-for a complete list.
-
-Sample Documents
-~~~~~~~~~~~~~~~~
-
-The examples in this guide use the following sample documents from the
-``test.pizza`` collection. The collection contains documents that
-describe the customers and their pizza orders as array elements in field
-called ``items``.
-
-.. code-block:: javascript
-
-   [{
-      name: "Steve Lobsters",
-      address: "731 Yexington Avenue",
-      items: [
-        { type: "beverage", name: "Water", size: "17oz", },
-        { type: "pizza", size: "large", toppings: ["pepperoni"], },
-        { type: "pizza", size: "medium", toppings: ["mushrooms", "sausage", "green peppers"], comment: "Extra green peppers please!", },
-        { type: "pizza", size: "large", toppings: ["pineapple, ham"], comment: "red pepper flakes on top", },
-        { type: "calzone", fillings: ["canadian bacon", "sausage", "onion"], },
-        { type: "beverage", name: "Diet Pepsi", size: "16oz", },
-      ],
-    },
-    {
-      name: "Popeye",
-      address: "1 Sweethaven",
-      items: [
-        { type: "pizza", size: "large", toppings: ["garlic, spinach"], },
-        { type: "calzone", toppings: ["ham"], },
-      ],
-    }]
+For a list of array update operators, see :manual:`Update Operators
+</reference/operator/update-array/>` in the Server
+Manual documentation.
 
 Specifying Array Elements
 -------------------------
 
-You can specify which array elements to update using a positional
-operator. Positional operators can specify the first, all, or certain
-array elements to update.
+Positional operators specify which array elements to update. Using these
+operators, you can apply updates to the first element, all elements, or
+certain elements that match a criteria.
```

Comment:
Suggestion:

Not a big deal, but I think this second sentence could be shorter and more direct:
"You can use these operators to apply updates to..."
Or add some specificity:
"You can call these operators to apply updates to..."

From the [style guide (use-effective-verbs)](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/):

> Replace weak verbs and gerunds with strong, action-oriented
> verbs to restore focus  to verbs from nouns. Relying on verbs rather
> than nouns usually makes sentences shorter, clearer, and more direct.